packages/@guidesmiths/cuckoojs-schematics/src/dockerfile/files/nestjs/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# -- BUILD STAGE --------------------------------
+FROM node:<%=nodeVersion%>-alpine as builder
+
+ENV NODE_ENV build
+
+USER node
+WORKDIR /home/node
+
+COPY package*.json ./
+RUN npm ci
+
+COPY --chown=node:node . .
+RUN npm run build \
+    && npm prune --production
+
+# -- PROD STAGE ----------------------------------
+FROM node:<%=nodeVersion%>-alpine
+
+ENV NODE_ENV production
+
+USER node
+WORKDIR /home/node
+
+COPY --from=builder --chown=node:node /home/node/package*.json ./
+COPY --from=builder --chown=node:node /home/node/node_modules/ ./node_modules/
+COPY --from=builder --chown=node:node /home/node/dist/ ./dist/
+
+CMD ["node", "dist/server.js"]
```

Comment:
Do we need to be able to configure the command to be run or will it be always the same for nodejs or nestjs apps?

Reply:
I'd keep it this way and we we start using it, we could iterate over this

Reply:
Sounds good!